modules/mono/editor/GodotTools/GodotTools/Build/BuildOutputView.cs

```diff
@@ -293,15 +293,15 @@ private void ScrollToLastNonEmptyLogLine()
         public void RestartBuild()
         {
             if (!HasBuildExited)
-                throw new InvalidOperationException("Build already started");
+                throw new InvalidOperationException("Build already started.");
 
             BuildManager.RestartBuild(this);
         }
 
         public void StopBuild()
         {
             if (!HasBuildExited)
```

Comment:
I'm new to this but shouldn't this be `HasBuildExited` instead of `!HasBuildExited`?  Won't an in-progress build have this value as false?
Just trying understand it.